lib/utils/patterns.ts

```diff
@@ -72,7 +73,7 @@ export function authenticate(options: {}): Promise<void> {
 }
 
 export function askLoginType() {
-	return form.ask({
+	return form.ask<'web' | 'register' | 'token' | 'credentials', 'list'>({
```

Comment:
How about also having this (`'web' | 'register' | 'token' | 'credentials'`) as a type?